add slowness warning to visualisation app (#47)
covid19-forecast-hub-europe/covid19-forecast-hub-europe-website: visualisation.Rmd

</p>

<div class="alert alert-warning">

>   <strong>Warning!</strong> The visualisation app is currently slow to load. We are working to fix this. In the meantime, waiting for a few minutes should make the app apear.

Reviewer: ```suggestion
  <strong>Warning!</strong> The visualisation app is currently slow to load. We are working to fix this. In the meantime, you may have to wait a few minutes until the app appears.
```
